cmd/frontend/internal/app/ui/handlers.go

```diff
@@ -49,6 +50,9 @@ type Metadata struct {
 	// Description is the description of the page for Twitter cards, OpenGraph,
 	// etc. e.g. "View this link in Sourcegraph Editor."
 	Description string
+
+	// ShowPreview toggles Twitter/Slack/OpenGraph/etc previews.
```

Comment:
```suggestion
	// ShowPreview controls whether or not OpenGraph/Twitter card/etc metadata is rendered.
```

(this is a more accurate statement since we can't control how Slack etc. actually display things, just what info we provide them)